controllers/horizontal_runner_autoscaler_webhook.go

```diff
@@ -324,7 +327,7 @@ func (autoscaler *HorizontalRunnerAutoscalerGitHubWebhook) Handle(w http.Respons
 	}
 
 	autoscaler.workerInit.Do(func() {
-		batchScaler := newBatchScaler(context.Background(), autoscaler.Client, autoscaler.Log)
+		batchScaler := newBatchScaler(context.Background(), autoscaler.Client, log)
```

Comment:
A batch scaler is not tied to a specific scaling event. That said, this might be wrong as it adds various log fields related to the first scaling event that triggered the batch scaler start up, to every other scaling event that this batch scaler processes! Can we perhaps revert this back to `autoscaler.Log` if you're okay?